packages/rum-recorder/src/domain/record/serialize.spec.ts

```diff
@@ -21,10 +21,10 @@ describe('serializeNodeWithId', () => {
       expect(map).toEqual({})
     })
 
-    it('sets ignored serialized node id to IGNORED_NODE_ID', () => {
+    it('does not serialize ignored nodes', () => {
```

Comment:
Was there a benefit for a feature we are not using?
do we have a test case covering the reordering of nodes with one ignored?
[node, ignored, node] -> [node, node, ignore]

Reply:
> Was there a benefit for a feature we are not using?
I don't think there was a real benefit. The mutation processing was more complex and needed to differentiate between ignored nodes and not-yet-serialized nodes, but I think it was an implementation detail more than a feature.

>  do we have a test case covering the reordering of nodes with one ignored?

I think this is covered by other test (in particular, [this one](https://github.com/DataDog/browser-sdk/blob/main/packages/rum-recorder/src/domain/record/mutationObserver.spec.ts#L492)), but it won't hurt to add a new one for this specific case. I'll add it.